### ISSUE TITLE
test_table_to_lineofbearing fails in Pro

### ISSUE THREAD

**Issue by mfunk** [Esri/military-tools-geoprocessing-toolbox#94]
test_table_to_lineofbearing fails in ArcGIS Pro 1.3 with the following message:

```
Testing Table To Line Of Bearing (Pro).
Running tool (Table To Line Of Bearing)
ERROR: Start Time: Tuesday, May 31, 2016 7:45:17 AM
Failed to execute. Parameters are not valid.
ERROR 000800: The value is not a member of 9102 | 9114 | 9101 | 9105.
ERROR 000800: The value is not a member of 9001 | 9036 | 9093 | 9030 | 9002 | 9003.
ERROR 000800: The value is not a member of 0 | 2 | 1 | 3.
ERROR 000800: The value is not a member of METERS | KILOMETERS | MILES | NAUTICAL_MILES | FEET | US_SURVEY_FEET.
ERROR 000800: The value is not a member of DEGREES | MILS | RADS | GRADS.
ERROR 000800: The value is not a member of GEODESIC | GREAT_CIRCLE | RHUMB_LINE | NORMAL_SECTION.
Failed to execute (TableToLOB).
Failed at Tuesday, May 31, 2016 7:45:18 AM (Elapsed Time: 0.11 seconds)
Start Time: Tuesday, May 31, 2016 7:45:17 AM
Failed to execute. Parameters are not valid.
ERROR 000800: The value is not a member of 9102 | 9114 | 9101 | 9105.
ERROR 000800: The value is not a member of 9001 | 9036 | 9093 | 9030 | 9002 | 9003.
ERROR 000800: The value is not a member of 0 | 2 | 1 | 3.
ERROR 000800: The value is not a member of METERS | KILOMETERS | MILES | NAUTICAL_MILES | FEET | US_SURVEY_FEET.
ERROR 000800: The value is not a member of DEGREES | MILS | RADS | GRADS.
ERROR 000800: The value is not a member of GEODESIC | GREAT_CIRCLE | RHUMB_LINE | NORMAL_SECTION.
Failed to execute (TableToLOB).
Failed at Tuesday, May 31, 2016 7:45:18 AM (Elapsed Time: 0.11 seconds)
```

**Comment by mfunk** [comment]
So the values are coming from core GP's **Bearing Distance To Line** tool, where the numbers you see listed are the index value for the esriSRUnitTypes: http://resources.esri.com/help/9.3/arcgisdesktop/arcobjects/esrigeometry/esrisrunittype.htm

So why is the tool asking for the index and not the unit name like the tool does in the dialog? We brought this to the GP team a while ago. Time to shake the tree again.

**Comment by mfunk** [comment]
Fixed in #146
